### DISCUSSION TITLE
Cannot login to admin page

### DISCUSSION THREAD

**Discussion by pcace** [ether/etherpad-lite#5140] [General]
Hi, 

i hope i am right here... 
i have just installed etherpad light via github, set the admin password in settings.json and restarted etherpad. (the pad itself works pretty good!)
the problem now is, that i cannot login to the admin page, neither with admin/changeme1 now with admin/mynewpassword it just brings up the loginpage again. 
Where could i start looking for the problem?

Thank you so much!

**Comment by webzwo0i** [comment]
Did you forget to remove `/*` and `*/` around the users section? By default the section is commented

**Comment by pcace** [comment]
embarrassing...
that solved it....
